.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Question
```

Comment:
Either this template or we can activate the "discussions" feature of GitHub. I am fine with either of them. I guess with the minimal expected traffic, just an issue template shall be enough.

Reply:
Agreed, issue template is enough for now. And most of the users would go to the issues instead of discussions anyway.